### ISSUE TITLE
Update docs for choosing a sidecar image

### ISSUE THREAD

**Issue by ghost** [eclipse-che/che#16631]
The docs for using a VSCode extension in Che recommend using the sidecar image `eclipse/che-theia-endpoint-runtime` for extensions with no external dependencies: https://www.eclipse.org/che/docs/che-7/using-a-visual-studio-code-extension-in-che/#choosing-the-sidecar-image_using-a-visual-studio-code-extension-in-che

The "che-theia-endpoint-runtime" Docker image was deprecated last October in [this che-theia commit](https://github.com/eclipse/che-theia/commit/df22b8912dabacee92ee221dc687f7d0bcc6ce21#diff-51447cd3c1f50513440c88549f4cbc33), and does not have a tag for Che versions after 7.3.

What is the latest recommendation for a sidecar image for extensions with no external dependencies?

**Comment by sleshchenko** [comment]
> What is the latest recommendation for a sidecar image for extensions with no external dependencies?

@AndrienkoAleksandr Could you provide some recommendations here?

**Comment by AndrienkoAleksandr** [comment]
Yes, it was deprecated because we don't want to force users to extends from some base image: https://github.com/eclipse/che-docs/pull/1071/files I am wondering why this docs is not visible on the https://www.eclipse.org/che/docs/che-7 and we have deprecated material:  https://www.eclipse.org/che/docs/che-7/using-a-visual-studio-code-extension-in-che/#choosing-the-sidecar-image_using-a-visual-studio-code-extension-in-che

**Comment by AndrienkoAleksandr** [comment]
Sorry, remote plugins is plenty important stuff, so I set up higher priority.

**Comment by ibuziuk** [comment]
Closing as deprecated
